platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/ClearTrigger.java

```diff
@@ -19,4 +19,9 @@
 /**
  * A placeholder object, to provide a type to the input `clear` wire for wiring components.
  */
-public record ClearTrigger() {}
+public final class ClearTrigger {
```

Comment:
This might be a good opportunity to just delete `ClearTrigger`, and use `NoInput`